packages/client/src/pages/forum-topic-page/forum-topic-page.tsx

```diff
@@ -0,0 +1,32 @@
+import { useParams } from 'react-router-dom';
+import { setPageTitle } from '@/helpers/helper';
+import { PageWrapper } from '@/components';
+import { FORUM_TOPICS_LIST } from '@/helpers/constants/forum';
+import { ForumTopic } from '@/components';
+import { ExitButton } from '@/components';
+
+export const ForumTopicPage = () => {
+  const { id } = useParams();
+  const topic = FORUM_TOPICS_LIST.find(
+    topic => topic.id === parseInt(id as string)
+  );
+
+  setPageTitle(topic ? topic.title : 'Форум');
```

Comment:
[Предложение]
Тут можно сократить запись с оператором `??`
```suggestion
  setPageTitle(topic?.title ?? 'Форум');
```

Reply:
Спасибо, заменил. Выглядит лаконичнее.